### PR TITLE
Improve CLI

### PR DESCRIPTION
**Pull request by LuKks**
Don't merge yet. We first need https://github.com/mafintosh/dht-rpc/pull/58 and https://github.com/holepunchto/hyperswarm-dht/pull/117 to later update this PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -269,15 +269,56 @@ See [dht-rpc](https://github.com/mafintosh/dht-rpc) for the additional APIs the
...
+
+2) Provide the first persistent node using your own bootstrap values:
+
+```sh
+hyperswarm-dht --port 49738 --bootstrap (server-ip):49737 --persistent
+```
+
+Important: it requires the port to be open like the bootstrap node.
+
+Don't use the `--persistent` flag again, or you could damage the network health.
```
Comment (mafintosh) [inline README.md:318]: lets not have the flag, footgun

**Comment by LuKks** [comment]
Btw this also fixes this case where incorrectly creates a bootstrap node:
`hyperswarm-dht --bootstrap <host:port> --host <ip> --port <n>`

File: README.md (hunk 1 above)
Comment (LuKks) [reply README.md:318]: 100%, just depends on https://github.com/mafintosh/dht-rpc/pull/58

**Comment by LuKks** [comment]
I still left the `--persistent` in the CLI but removed from the docs, it's actually useful when you're playing with own DHT networks but can't be recommended to use. Ideally the bootstrap node should serve for routing peers as per https://github.com/mafintosh/dht-rpc/pull/58

File: bin.js [hunk 2]
```diff
@@ -21,32 +23,55 @@ function arg (name) {
...
+    })
+
+    node.on('close', function () {
+      console.log('Node #' + id + ' closed')
     })
   }
 
   console.log('Fully started ' + cnt + ' Hyperswarm DHT node' + (cnt === 1 ? '' : 's'))
+
+  process.once('SIGINT', function () {
```
Comment (mafintosh) [inline bin.js:72]: you should print here, "Shutting down nodes..."

**Pull request merged**